communitydays/index.html

```diff
@@ -491,18 +491,18 @@ <h4>Speaker: Dr. Salva Daneshgadeh Cakmakci</h4>
-                    Michael will discuss the challenges of consuming and publishing updates from different perspectives within the supply chain, covering roles such as:
+                    Michael Pfurtscheller will discuss the challenges of consuming and publishing updates from different perspectives within the supply chain, covering roles such as:
                     <ul>
                         <li>Hardware or Software Component Developers</li>
                         <li>Product & Device Manufacturers</li>
                         <li>Solution and Application Developers</li>
                         <li>Product Integrators like Factory Operators, Car Manufacturers, etc.</li>
                     </ul>
                  </p>
-                <p><strong>Bio Tobi:</strong> Tobi has been in the security field for 20 years, with over a decade of experience focused on the industrial side of IT infrastructures. He began his journey with vulnerability handling at Siemens ProductCERT, where he played a key role in automating security tests. His current research areas include tool-based vulnerability management and risk-based mitigation decisions.</p>
-                <p><strong>Bio Michael:</strong> Michael is a Product Security Manager at u-blox AG but has mainly worked as a consultant since 1998. Security was always part of the job when working in the fields of SAP programming, management of messaging and collaboration infrastructures, network and computing center infrastructures, e-commerce projects and Pentesting. Currently, he focuses on establishing an ISO 21434-compliant Cybersecurity Management System (CSMS) for u-blox and its GNSS and IoT products, including vulnerability, incident, and update management for its modules and firmware.</p>
+                <p><strong>Bio Tobias Limmer:</strong> Tobias Limmer has been in the security field for 20 years, with over a decade of experience focused on the industrial side of IT infrastructures. He began his journey with vulnerability handling at Siemens ProductCERT, where he played a key role in automating security tests. His current research areas include tool-based vulnerability management and risk-based mitigation decisions.</p>
+                <p><strong>Bio Michael Pfurtscheller:</strong> Michael Pfurtscheller is a Product Security Manager at u-blox AG but has mainly worked as a consultant since 1998. Security was always part of the job when working in the fields of SAP programming, management of messaging and collaboration infrastructures, network and computing center infrastructures, e-commerce projects and Pentesting. Currently, he focuses on establishing an ISO 21434-compliant Cybersecurity Management System (CSMS) for u-blox and its GNSS and IoT products, including vulnerability, incident, and update management for its modules and firmware.</p>
```

Comment:
```suggestion
                <p><strong>Bio Michael Pfurtscheller:</strong> Michael Pfurtscheller is the Product Security Manager at u-blox AG but has mainly worked as a consultant since 1998. Security was always part of the job when working in the fields of SAP programming, management of messaging and collaboration infrastructures, network and computing center infrastructures, e-commerce projects and Pentesting. Currently, he focuses on establishing an ISO 21434-compliant Cybersecurity Management System (CSMS) for u-blox and its GNSS and IoT products, including vulnerability, incident, and update management for its modules and firmware.</p>